data_rip.sh

```diff
@@ -10,15 +10,16 @@ source "$ARM_CONFIG"
 
 
         TIMESTAMP=$(date '+%Y%m%d_%H%M%S');
-        DEST="/mnt/media/ARM/${TIMESTAMP}_${ID_FS_LABEL}"
-        mkdir "$DEST"
+        DEST="/mnt/media/ARM/Media/Data/${TIMESTAMP}_${ID_FS_LABEL}"
+        mkdir -p "$DEST"
 	FILENAME=${ID_FS_LABEL}_disc.iso
 
 
 	#dd if=/dev/sr0 of=$DEST/$FILENAME 
 	cat "$DEVNAME" > "$DEST/$FILENAME"
 
 	eject
-
+	
+	chmod 777 -R "$DEST"
```

Comment:
Some people with a NAS may not want to set it to 777.  Perhaps add an option in the config file, maybe "SET_MEDIA_PERMISSIONS="true" / "false" ?